src/db/tables/attributions.ts

```diff
@@ -60,7 +60,10 @@ async function addAttributionForSubscriber(
     res = await knex("attributions").insert(attribution).returning("*");
   } catch (e) {
     if ((e as Error).message.includes("violates unique constraint")) {
-      logger.info("Ignoring unique constraints");
+      logger.info("addAttributionForSubscriber", {
+        subscriberId,
+        error: (e as Error).message,
```

Comment:
Good catch, we have a bunch of places where we're not logging the exception very effectively. I was thinking about trying to centralize this and have an easy way to log the message and the stack.

Reply:
We can create a ticket to track this

Reply:
Created https://mozilla-hub.atlassian.net/browse/MNTOR-2914